devices/oneplus-enchilada/default.nix

```diff
@@ -0,0 +1,126 @@
+{ config, lib, pkgs, ... }:
+
+{
+  mobile.device.name = "oneplus-enchilada";
+  mobile.device.identity = {
+    name = "OnePlus 6";
+    manufacturer = "OnePlus";
+  };
+
+  mobile.hardware = {
+    soc = "qualcomm-sdm845";
+    ram = 1024 * 8;
+    screen = {
+      width = 1080; height = 2280;
+    };
+  };
+
+  mobile.boot.stage-1 = {
+    compression = "xz";
+    kernel.package = pkgs.callPackage ./kernel { };
+    firmware = [
+      config.mobile.device.firmware
+    ];
+  };
+
+
+  mobile.device.firmware = pkgs.callPackage ./firmware {};
+
+  mobile.system.android.device_name = "OnePlus6";
+  mobile.system.android = {
+    ab_partitions = true;
+    bootimg.flash = {
+      offset_base = "0x00000000";
+      offset_kernel = "0x00008000";
+      offset_ramdisk = "0x01000000";
+      offset_second = "0x00f00000";
+      offset_tags = "0x00000100";
+      pagesize = "4096";
+    };
+  };
+
+  boot.kernelParams = [
+    "console=tty0"
+    "panic=10"
```

Comment:
The default should be set as [`CONFIG_PANIC_TIMEOUT`](https://cateee.net/lkddb/web-lkddb/PANIC_TIMEOUT.html).

(Not for you to do) this should be added to the mandatory validated kernel config.